mods/ra/audio/notifications.yaml

```diff
@@ -113,7 +113,9 @@ Speech:
 		WarningFourMinutesRemaining: 4minr
 		WarningFiveMinutesRemaining: 5minr
 		Win: misnwon1
-
+	AltTexts:
```

Comment:
You can disable the automatic FieldLoader parsing with `[Fieldloader.Ignore]`, and then manually enumerate over the keys to parse the `node.Value.Value` and `node.Nodes.FirstOrDefault(n => n.Key == "TextNotification")`.